src/buffer/out/textBuffer.cpp

```diff
@@ -2265,6 +2267,48 @@ HRESULT TextBuffer::Reflow(TextBuffer& oldBuffer,
             CATCH_RETURN();
         }
 
+        // GH#32: Copy the attributes from the rest of the row into this new buffer.
+        // From where we are in the old buffer, to the end of the row, copy the
+        // remaining attributes.
+        // - if the old buffer is smaller than the new buffer, then just copy

```

Comment:
Add a comment explaining "we already copied all text with the attribute color, but it's possible for someone to just put some color into the buffer to the right of that without any text. the buffer looks weird to the user when we resize and it starts losing those colors, so we need to copy them over too... as long as there is space."

Reply:
aka some sort of "plain english" explanation without requiring someone to understand "attributes == colors"